src/views/workflow-actions/config/workflow-actions.config.ts

```diff
@@ -48,6 +54,23 @@ const workflowActionsConfig: [
     apiRoute: 'terminate',
     getSuccessMessage: () => 'Workflow has been terminated.',
   },
+  {
+    id: 'restart',
+    label: 'Restart',
+    subtitle: 'Restart a workflow execution',
+    modal: {
+      text: [
+        'Restarts a workflow by creating a new execution with a fresh Run ID while using the existing input. If the previous execution is still running, it will be terminated.',
+        'What differentiates Restart from Reset is that restarted workflow is not aware of the previous workflow execution.',
```

Comment:
```suggestion
        'What differentiates Restart from Reset is that a restarted workflow is not aware of the previous workflow execution.',
```
nit: you could also use "the"